src/main/java/frc/robot/Subsystems/VisionSubsystem.java

```diff
@@ -46,7 +47,7 @@ public class VisionSubsystem extends SubsystemBase {
     );
 
     public VisionSubsystem() {
-        camera = new PhotonCamera("lol");
+        camera = new PhotonCamera("IMX219");
```

Comment:
Did we change the name or is this a new camera?

Reply:
we changed the name